src/services/Rpc/index.js

```diff
@@ -0,0 +1,21 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import * as React from 'react';
+
+class Rpc extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { data: {} };
+      }
+
+      componentDidMount() {
+        this.init();
+      }
+
+      init = async () => {
+        const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+        const api = await ApiPromise.create({ provider: wsProvider });
+        return api;
+      }
+}
+
+export default Rpc;
```

Comment:
1. using functional components + hooks instead of class components
2. services should be regular classes, not wrapped inside a react component

Reply:
@isaacadams Regarding #2, can you point me to an example of this somewhere?

Reply:
There's a "converting a function to a class" section but its just a React.Component: https://reactjs.org/docs/state-and-lifecycle.html

Reply:
Unless I'm overthinking this and you mean a literal JavaScript class?

Reply:
yes, a literal javascript class

I will provide a POC tomorrow which shows how you will be able to get state in a regular javascript class using RxJS library